src/enochecker/enochecker.py

```diff
@@ -490,7 +490,18 @@ def run(self, method: Optional[Union[str, Callable]] = None) -> CheckerResult:
                 ),
                 exc_info=eno,
             )
-            return CheckerResult.from_exception(eno)  # , eno.message
+
+            if (
+                self.flag
+                and self.flag in str(eno)
+                or (eno.message and self.flag in eno.message)
+            ):
+                self.error(f"EnoMessage contained flag! (Exception was {eno}")
+                eno.message = "Something went wrong"
```

Comment:
Again, do we really need this message here? This doesn't really help the ctf players.

Reply:
This was meant more as a warning to us ("oh, this message is new?") but yeah not really player facing.. removed